string mode
int64 T_home
---
string home
